tensorhive/controllers/restriction/remove_from_restriction.py

```diff
@@ -0,0 +1,129 @@
+from tensorhive.authorization import admin_required
+from tensorhive.exceptions.InvalidRequestException import InvalidRequestException
+from sqlalchemy.orm.exc import NoResultFound
+from tensorhive.models.Restriction import Restriction
+from tensorhive.models.User import User
+from tensorhive.models.Group import Group
+from tensorhive.models.Resource import Resource
+from tensorhive.models.RestrictionSchedule import RestrictionSchedule
+from tensorhive.config import API
+import logging
+log = logging.getLogger(__name__)
+R = API.RESPONSES['restriction']
+U = API.RESPONSES['user']
+GROUP = API.RESPONSES['group']
+RESOURCE = API.RESPONSES['resource']
+S = API.RESPONSES['schedule']
+G = API.RESPONSES['general']
+
+
+@admin_required
+def remove_from_user(restriction_id, user_id):
+    try:
+        restriction_not_found = True
```

Comment:
same as before... check if `restriction is None` (same applies to subsequent methods in this file)